### PR TITLE
Improve versions UI

### PR DESCRIPTION
**Pull request by houmark**
This PR improves the usability of the versions modal.

With this PR Apostrophe will now allow the user to click on the same "See Changes" link to close a version change, which is nice when comparing many and long changes.

Additionally a default message is shown if there is no changes to display. This message is added in the template and fetched in JS to make it localizable.

### CODE REVIEW

**Comment by houmark** [comment]
This one is gonna need an improvement. I'll be working on that and let you know when it's ready for testing.

**Comment by houmark** [comment]
Nah, this one is good to go. I am removing content in the html container and thought that if the next click is not ajax'ed, then it will stay empty, but it should do that, so no problem.

One could show and hide it after the inital load and then cache the first ajax, but that would become more complex, so I think this is fine to start with (keeping this open for additional improvements in the future).

File: lib/modules/apostrophe-versions/public/js/editor.js [hunk 1]
```diff
@@ -19,9 +19,18 @@ apos.define('apostrophe-versions-editor', {
         var $version = $toggle.closest('[data-version]');
         var currentId = $version.attr('data-version');
         var oldId = $version.attr('data-previous');
-        self.html('compare', { oldId: oldId, currentId: currentId }, function(html) {
-          $version.find('[data-changes]').html(html);
-        });
+        var $versionContent = $version.find('[data-changes]');
+        var noChanges = $('[data-no-changes]').attr('data-no-changes');
```
[the remark this answers is not in the document]
Comment (houmark) [reply lib/modules/apostrophe-versions/public/js/editor.js]: Correct, sorry about that one! I guess I thought that this being ever only once in the page a global selector would be fine, but I will keep it in mind in the future. Pushed a new commit that corrects to use `self.$el.find`.

**Comment by boutell** [comment]
Thanks!

**Pull request merged**